src/cmd/report.go

```diff
@@ -73,11 +74,11 @@ type brewResp struct {
 	Items      []formulaMetric `json:"items"`
 }
 
-func fetchBrewData(formula string) {
+func reportBrewData(formula string) {
 
 	resp, err := http.Get("https://formulae.brew.sh/api/analytics/install-on-request/30d.json")
 	if err != nil {
-		log.Fatal("Error: Failed to fetch Brew data.")
+		log.Fatal("Error: Failed to report Brew data.")
```

Comment:
These functions should still be fetch as they is what they are doing. The command is reporting, not these functions.